tests/queries/0_stateless/01764_prefer_column_name_to_alias.sql

```diff
@@ -0,0 +1,3 @@
+SELECT avg(number) AS number, max(number) FROM numbers(10) settings prefer_column_name_to_alias  = 1;
```

Comment:
Please, add the same queries without this settings - to see that it really makes a difference.

Reply:
Done.